### PR TITLE
Adding security issue response blogpost

### LINKED ISSUE #983: Blogpost about new security issue response process
**Describe the blog post your would like to write**
A short blogpost to announce the PR about the new security issue response process.

**What is the title of the blog post?**
TBD (

**Who are the authors?**
Dave Lago

**What is the proposed posting date?**
9/16

### PR DESCRIPTION
**Pull request by davidlago**
Signed-off-by: Dave Lago <davelago@amazon.com>

### Description
A short blogpost to announce the PR about the new security issue response process.
 
### Issues Resolved
Closes #983 

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2022-09-16-security-issue-response.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title:  "Documenting our Security Issue Response Process"
```
Comment (natebower) [inline _posts/2022-09-16-security-issue-response.md:3]: Sentence case: Documenting our security issue response process

File: _posts/2022-09-16-security-issue-response.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+authors:
+  - davidlago
+date:   2022-09-16 00:00:00 -0700
+categories:
+  - technical-post
+---
+
+Today we are taking the next step in our open source journey by updating our security policy to include a process for how we respond to security issues. In proper open source fashion, we are creating this as a [pull request](https://github.com/opensearch-project/.github/pull/XXX) and we are inviting everyone to take part in the discussion.
```
Comment (natebower) [inline _posts/2022-09-16-security-issue-response.md:11]: Hyphenate both instances of "open-source". Add a comma after the parenthetical.

File: _posts/2022-09-16-security-issue-response.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Besides making it transparent what the process looks like for responding to these issues, this policy has a significant addition: **the creation of a pre-disclosure list**.
```
Comment (natebower) [inline _posts/2022-09-16-security-issue-response.md:13]: Change to "Besides making the security issue response process transparent..."

File: _posts/2022-09-16-security-issue-response.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+Having a pre-disclosure list (a group of individuals and companies that will be informed of a security issue before it becomes public) is a risk balancing act. If a security issue gets pre-disclosed to too broad of an audience, we risk it becoming public ahead of time. On the other hand, we might be missing out on collaborations to expedite the creation and testing of a fix by not having a mechanism for involving some individuals and companies early, in addition to effectively creating zero-days for all the issues we disclose, leaving everyone scrambling to apply the fixes.
```
Comment (natebower) [inline _posts/2022-09-16-security-issue-response.md:15]: Change "gets" to "is". Change "collaborations to" to "collaborations that would".

File: _authors/davidlago.markdown [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+name: Dave Lago
+short_name: davidlago
+photo: '/assets/media/authors/davidlago.jpg'
+github: davidlago
+---
+
+**Dave Lago** is a Software Development Manager at AWS working on the OpenSearch project.
```
Comment (natebower) [inline _authors/davidlago.markdown:8]: This might be smoother as "Dave Lago is an AWS Software Development Manager working on the OpenSearch project."

**Review by natebower — COMMENTED** [review]
@davidlago Please see my comments and let me know if you have any questions. Thanks!

**Comment by davidlago** [comment]
@natebower comments addressed. Thanks!

**Comment by davidlago** [comment]
FYI this PR is not to be merged until the related PR is posted and I update the link here.

**Comment by davidlago** [comment]
Good to merge! Linked PR has been posted and the text updated with the link 👍

File: _authors/davidlago.markdown [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
+name: Dave Lago
+short_name: davidlago
+photo: '/assets/media/authors/davidlago.jpg'
```
Comment (krisfreedain) [inline _authors/davidlago.markdown:4]: looks like this should be .png instead of .jpg

**Review by krisfreedain — CHANGES_REQUESTED** [review]
@davidlago - one tiny change required

**Pull request merged**